wazuh/wazuh-dashboard-plugins
# Fix incorrect red status for up to date agent versions

### Description
This PR addresses the issue where the agent version status is incorrectly displayed as red (outdated) even when the agent is actually up to date. 
 
### Issues Resolved
https://github.com/wazuh/wazuh-dashboard-plugins/issues/6866

### Evidence
![image](https://github.com/user-attachments/assets/4c598f85-a594-48b6-a806-3b1ede0cc302)

### Test

1. Add up to date agents, outdated agents and never connected agents.
2. Go to the Endpoints Summary.
3. Verify the agent version status.

### Check List
